pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,8 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		commit := fmt.Sprintf("%s", rev.Commit[:(len(rev.Commit)/3)])
```

Comment:
i agree with @liggitt.  Let's just do the if.
if len>7  do :7 excerpt
else
  use full value provided.

or if you want to get fancy, rev.Commit[min(7,len(rev.Commit)]